src/app/shared/doc-viewer/docs-api-module-import/docs-api-module-import.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, AfterViewInit, OnInit } from "@angular/core";
+import { MatSnackBar } from "@angular/material/snack-bar";
+import { CopierService } from "../../copier/copier.service";
+
+const EXPORT_NAME_PLACEHOLDER = "{{exportName}}";
+const MODULE_IMPORT_PATH_PLACEHOLDER = "{{moduleImportPath}}";
+const MODULE_IMPORT_TEMPLATE = `import ${EXPORT_NAME_PLACEHOLDER} from '${MODULE_IMPORT_PATH_PLACEHOLDER}';`;
+
+@Component({
+  selector: "docs-api-module-import",
+  template: `
+    <p class="docs-api-module-import">
+      <code>
+        {{ moduleImport }}
+      </code>
+    </p>
+    <button
+      class="docs-api-module-import-copy"
+      mat-icon-button
+      (click)="copyModuleImport()"
+      matTooltip="Copy module import">
+      <span class="material-icons">file_copy</span>
+    </button>
+  `,
+})
+export class DocsApiModuleImport implements OnInit {
+  @Input() exportName: string;
+  @Input() moduleImportPath: string;
+
+  moduleImport: string;
+
+  constructor(private snackbar: MatSnackBar, private copier: CopierService) {}
+
+  ngOnInit(): void {
+    this.moduleImport = MODULE_IMPORT_TEMPLATE.replace(
+      EXPORT_NAME_PLACEHOLDER,
+      this.exportName
+    ).replace(MODULE_IMPORT_PATH_PLACEHOLDER, this.moduleImportPath);
+  }
+
+  copyModuleImport(): void {
+    if (this.copier.copyText(this.moduleImport)) {
+      this.snackbar.open("Module import copied", "", { duration: 2500 });
```

Comment:
Please run `npm run lint` and correct the issues with double quotes. You may be able to auto correct some of them via `ng lint --fix`.

Reply:
haven't noticed the double quotes. well, that's a shame on my side